packages/mux-player/README.md

```diff
@@ -117,6 +117,50 @@ Enable the [Google Cast](https://developers.google.com/cast) button in the contr
 <script defer src="https://www.gstatic.com/cv/js/sender/v1/cast_sender.js?loadCastFramework=1"></script>
 ```
 
+### CSS Parts
+
+Mux Player uses a [shadow DOM](https://developer.mozilla.org/en-US/docs/Web/Web_Components/Using_shadow_DOM)
+to encapsulate its styles and behaviors.
+As a result, it's not possible to target its internals with the usual CSS selectors.
+Instead, some components expose **parts** that can be targeted with the [CSS part selector](https://developer.mozilla.org/en-US/docs/Web/CSS/::part)
+, or `::part()`.
+
+```html
+<style>
+  mux-player::part(center play button) {
+    display: none;
+  }
+</style>
+<mux-player playback-id="DS00Spx1CV902MCtPj5WknGlR102V5HFkDe"></mux-player>
+```
+
+Supported **parts**:
```

Comment:
suggestion(non-blocking): This is a good first pass, but either now or as a followup we should discuss how these parts relate to the controls and "regions" of the UI